src/toil/wdl/wdl_interpret.py

```diff
@@ -0,0 +1,855 @@
+# Copyright (C) 2018 UCSC Computational Genomics Lab
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+
+import json
+import csv
+import os
+import logging
+
+import toil.wdl.wdl_parser as wdl_parser
+
+wdllogger = logging.getLogger(__name__)
+
+
+class InterpretWDL:
+    '''
+    A program to run WDL input files using native Toil scripts.
```

Comment:
IMO, like I mentioned above, CompileWDL and InterpretWDL should have different docstrings--doesn't need to be anything too complex, but it would go a long way toward clarifying what the difference is.